toolkit/scripts/srpm_pack.mk

```diff
@@ -58,8 +61,27 @@ $(STATUS_FLAGS_DIR)/build_srpms.flag: $(local_specs) $(local_spec_dirs) $(SPECS_
 			$(call print_error,Failed to download $${srpm_file});  } \
 	done || $(call print_error,Loop in $@ failed) ; \
 	touch $@
+
+$(STATUS_FLAGS_DIR)/build_toolchain_srpms.flag: $(STATUS_FLAGS_DIR)/build_srpms.flag
```

Comment:
This took me a second to figure out, could you add a comment on why this works?

Reply:
(ie, since we are downloading everything we don't need to differentiate)

Reply:
On second thought, should these SRPMs really co-exist with the normal ones in the same folder?

I would tend to put them in their own folder (`$(BUILD_TOOLCHAIN_SRPMS_DIR): $(STATUS_FLAGS_DIR)/build_toolchain_srpms.flag`)

Reply:
Maybe it could even be a sub-folder of the normal specs dir... thinking about the impacts on publishing the SRPMs.